cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -139,6 +142,39 @@ public void onSuccess(String response) {
             banners.add(banner);
           }
 
+          List<Banner> bannerList = new ArrayList<>(banners);
+          Collections.sort(bannerList, new Comparator<Banner>() {
+            @Override
+            public int compare(Banner banner1, Banner banner2) {
+              int result;
+              if (banner1.getStartAt() == null || banner2.getStartAt() == null) {
+                return 0;
+              }
+              Date date1 = banner1.getStartAt();
+              Date date2 = banner2.getStartAt();
+              Calendar calendar1 = Calendar.getInstance();
+              Calendar calendar2 = Calendar.getInstance();
+              calendar1.setTime(date1);
+              calendar2.setTime(date2);
+              calendar1.set(Calendar.MILLISECOND, 0);
+              calendar1.set(Calendar.SECOND, 0);
+              calendar2.set(Calendar.MILLISECOND, 0);
+              calendar2.set(Calendar.SECOND, 0);
+              if (calendar2.equals(calendar1)) {
```

Comment:
we can do 
result = calendar2.compareTo(calendar1);
instead of
the whole if else condition or?